app/src/features/projects/ProjectPage.tsx

```diff
@@ -96,7 +96,10 @@ const ProjectPage: React.FC = () => {
       setOpenErrorDialogProps({ ...openErrorDialogProps, ...textDialogProps, open: true });
     };
 
-    if (files && files.length > 0) {
+    if (!files) {
+      setDropzoneText('Select files');
+      return;
+    } else {
```

Comment:
Looks good, but we can actually now get rid of the `else` entirely, since if we go into the `if` we will return, and if we dont go into the `if` we will continue and hit the happy path code.  So the `else` doesnt actually do anything (which is good, one more level of nesting we can remove).

Reply:
right